adserver/templates/adserver/advertiser/advertisement-create.html

```diff
@@ -27,15 +27,27 @@ <h1>{% block heading %}{% trans 'Create advertisement' %}{% endblock heading %}<
   </div>
 
   <div class="col-md ml-md-5">
+    <div class="mb-5">
+      {% if ad_types %}
+        <h5>{% trans 'Ad types' %}</h5>
+
+        {% for ad_type in ad_types %}
+          <div class="mt-4">
+            <h6>{{ ad_type.name }}</h6>
+            <p class="small">{{ ad_type.description }}</p>
+          </div>
+        {% endfor %}
+      {% endif %}
+    </div>
+
     {% if ad_types %}
-      <h5>{% trans 'Ad types' %}</h5>
+    <div class="ad-previews" data-bind="visible: totalLength() > 10">
```

Comment:
Do we need to hide this below 10 chars? Seems like it might be jarring to have it display randomly, but not 100% sure.

Reply:
In testing, I thought it improved things. It's very odd to have a preview with nothing there.

Reply:
Cool, just confirming 👍